lib/index.js

```diff
@@ -140,6 +140,14 @@ function getOptions(options) {
   options.sourceMap = getSourceMap(options);
   options.style = getStyle(options) || 0;
 
+  // context object represents node-sass environment
+  options.context = {
+    info: module.exports.info,
+    state: {},
+    sync: false,
```

Comment:
Why do we need these three? `info` and `sync` are useless IMO, `state` can be set by user if they need. //cc @kevva.